### DISCUSSION TITLE
Dockge container is creating files owned by root user.

### DISCUSSION THREAD

**Discussion by queeup** [louislam/dockge#348] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I am using compose file to deploy Dockge with my normal user (not root).  If I create compose file with Dockge, it creates root owned compose.yaml on my system. I expected to create my users ig:group (1000:100) owned files.

### 🐻 Dockge Version

1.4.1

### 💻 Operating System and Arch

Nixos

### 🌐 Browser

Firefox 121

### 🐋 Docker Version

24.0.5

**Comment by trythatagain** [comment, marked as answer]
Please see here for a similar message [#89](https://github.com/louislam/dockge/discussions/89)
